embedding.go

```diff
@@ -150,7 +150,10 @@ func (l *Embedding) Fwd(a G.Input) G.Result {
 
 	if !l.inputIsOneHot {
 		oh = l.oh
-		l.Run(a.Node())
+		err := l.Run(a.Node())
```

Comment:
:+1:

Reply:
my sloppy code horrifies me. Thank for fixing

Reply:
errcheck :)